docs/using-docker.rst

```diff
@@ -10,22 +10,20 @@ Install Docker onto the machine you plan to run Teleirc from.
 Which image do I choose?
 ************************
 
-Node Slim, Official Node Alpine Linux, and Fedora images are provided (ordered ascending by size).
+Official Node Alpine Linux, and Fedora images are provided (ordered ascending by size).
 
 +-----------------------------------------------------------------------------+-----------------------+---------+
 | Image                                                                       | File                  | Size    |
 +=============================================================================+=======================+=========+
-| `Node Alpine Linux <https://hub.docker.com/r/_/node/>`_ (``node:6-alpine``) | ``Dockerfile.alpine`` | 66.9 MB |
-+-----------------------------------------------------------------------------+-----------------------+---------+
-| `Node Slim <https://hub.docker.com/r/_/node/>`_  (``node:6-slim``)          | ``Dockerfile.slim``   | 256 MB  |
+| `Node Alpine Linux <https://hub.docker.com/r/_/node/>`_ (``node:8-alpine``) | ``Dockerfile.alpine`` | 374 MB  |
```

Comment:
Just a random catch, is 374MB the correct image size for the Node 8 Alpine image? Seems like a big jump from 66.9MB, so I wanted to make sure.

Reply:
This is the overall teleirc image size -- not the base size.

Reply:
@robbyoconnor Ah, good to know. Thanks for clarifying.